tests/helpers.js

```diff
@@ -36,3 +39,17 @@ export function extractIfEnveloped(input) {
     return input;
   }
 }
+
+export async function addJsonAttachment(fileName, content) {
+  try {
+    // Temporarily disable the console log to avoid unnecessary info logs.
+    const consoleLog = console.log;
```

Comment:
@PatStLouis so not sure if we're all 100% agreeing on how to solve this, but do you think this will work:

1. create a separate command for allure reports
2. that separate command can turn off the consoleLog if needed

LMK if that will work.